From 5cdc07bbb92fbdb8830d2406771c6fd9a4593370 Mon Sep 17 00:00:00 2001
From: me-cafebabe <me_cafebabe@hotmail.com>
Date: Sat, 20 Aug 2022 15:43:00 +0000
Subject: [PATCH] nqnfcinfo: Use qti_kernel_headers

I'm not gonna import lineage kernel headers generator
and Blueprint module can't depend on makefile module

Change-Id: I5694c4fe8b595ee69e78d03590ae2f0dda9c4e34
---
 nqnfcinfo/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nqnfcinfo/Android.bp b/nqnfcinfo/Android.bp
index 6cc1c9d..e6c368e 100644
--- a/nqnfcinfo/Android.bp
+++ b/nqnfcinfo/Android.bp
@@ -12,7 +12,7 @@ cc_binary {
         "main.cpp",
     ],
 
-    header_libs: ["generated_kernel_headers"],
+    header_libs: ["qti_kernel_headers"],
 
     shared_libs: [
         "libc",
-- 
2.30.2

